web_monitoring/html_diff_render.py

```diff
@@ -249,13 +248,6 @@ def html_diff_render(a_text, b_text, a_headers=None, b_headers=None,
     text2 = '<!DOCTYPE html><html><head></head><body><h1>Header</h1></body></html>'
     test_diff_render = html_diff_render(text1,text2)
     """
-    raise_if_not_diffable_html(
-        a_text,
-        b_text,
-        a_headers,
-        b_headers,
-        content_type_options)
-
```

Comment:
On Slack, we talked specifically about still needing to do checks here: https://archivers.slack.com/archives/C47DJK8H0/p1559529058016900

At the server level, we can check whether the content is probably a binary type when trying to decode it to text, but we should still be doing additional checks here to determine whether we’re actually dealing with HTML-formatted text here (as opposed to, say, plain text or maybe a CSV or JSON document).